src/app/query/bitcoin/ordinals/inscriptions.utils.ts

```diff
@@ -0,0 +1,3 @@
+export function hasInscriptions(data: unknown[]) {
+  return data.length !== 0;
+}
```

Comment:
Nice utility, but can't help but feel this would better as two parts (as the logic isn't really specific to inscriptions)

```ts
// in common helper file
function isEmptyArray(data: unknown[]) {
  return data.length === 0;
}

function hasInscriptions(inscriptions: Inscription[]) {
  return !isEmptyArray(inscriptions);
}
```

Reply:
Yeah, good point, I just renamed the file but will look at changing.

Reply:
I really don't understand this original function bc it actually passes in `UtxoResponseItem[]` not inscriptions? 🤔 Was originally here: https://github.com/leather-wallet/extension/blob/4a4185a1eab0e550662d8f1cf93cb37fa5d98c49/src/app/query/bitcoin/ordinals/use-taproot-address-utxos.query.ts#L47